frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -446,6 +447,18 @@ export function PollingStationFormController({
     status.current = "deleted";
   };
 
+  const finaliseDataEntry = async () => {
+    status.current = "finalising";
+    const response = await client.postRequest(request_path + "/finalise");
+    status.current = "idle";
+    if (response.status !== ApiResponseStatus.Success) {
+      console.error("Failed to finalise data entry", response);
+      status.current = "idle";
```

Comment:
This is not needed, since the `status.current = "idle"` is already done on line 453, or that one has to be removed and line 456 kept.